LinqToRest/Async/Infrastructure/FakeAsyncEnumerable.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Messerli.LinqToRest.Async.Infrastructure
+{
+    public class FakeAsyncEnumerable<T> : IAsyncEnumerable<T>, IEnumerable<T>
+    {
+        private readonly IEnumerable<T> _source;
+
+        public FakeAsyncEnumerable(IEnumerable<T> source)
+        {
+            _source = source;
+        }
+
+        public async IAsyncEnumerator<T> GetAsyncEnumerator(CancellationToken cancellationToken = new CancellationToken())
+        {
+            foreach (var item in _source)
+            {
+                yield return item;
+            }
+
+            await Task.CompletedTask;
+        }
+
+        public IEnumerator<T> GetEnumerator()
```

Comment:
Use an expression body

Reply:
Fixed in f86d9cc2f36b5e9b66da30226c82ea0a5ada1b24